helm/production-values.yaml

```diff
@@ -0,0 +1,5 @@
+environment: production
+containers:
+  service:
+    registry: 'minesupport.azurecr.io/'   
+    apiGateway: http://mine-support-api-gateway:3001

```

Comment:
Long term I'd be tempted to split the port number into a separate value as the port number may be passed from a parent config that wires the services together. Don't think it's necessary now, though.

Reply:
Agreed.  Inclusion of http:// may also warrant further thinking.

Reply:
API Gateway port moved to new variable.